tests/Countries/ChileTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Spatie\Holidays\Tests\Countries;
+
+use Carbon\CarbonImmutable;
+use Spatie\Holidays\Holidays;
+
+it('can calculate Chile holidays', function () {
```

Comment:
Could you run the tests locally too and verify if the output is correct?

Reply:
No worries. :-)

Make sure your fork is synced with this repo: 
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork

Git rebase integrates changes from the main branch into your branch by applying your branch's commits on top of the latest main branch commits, resulting in a linear history. How to do this, might be best to google yourself.

Running the tests locally should generate a snapshot file. You'll have to commit that file too.